src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/packageScanner.hpp

```diff
@@ -189,9 +184,23 @@ class TPackageScanner final : public AbstractHandler<std::shared_ptr<TScanContex
             }
         };
 
-        const auto cnaName {m_databaseFeedManager->getCnaName(data->packageVendor(), data->packageFormat())};
+        auto cnaName {m_databaseFeedManager->getCnaNameByFormat(data->packageFormat().data())};
 
         if (cnaName.empty())
+        {
+            cnaName = m_databaseFeedManager->getCnaNameByPrefix(data->packageVendor().data());
+            if (cnaName.empty())
+            {
+                cnaName = m_databaseFeedManager->getCnaNameByContains(data->packageVendor().data());
+                if (cnaName.empty())
+                {
+                    cnaName = DEFAULT_CNA;
+                }
+            }
+        }
+
+        // TODO: remove this section when PyPi and NPM vendors are enabled
```

Comment:
Remove temporal code, the release is blocked by the issue of pypy and npm.

Reply:
Great, thank you.